mqtt/mqtt-broker-tests-util/src/client.rs

```diff
@@ -299,9 +299,9 @@ where
     }
 }
 
-pub fn create_client_from_module_env(address: String) -> Result<Client<ClientIoSource>, VarError> {
```

Comment:
I think it was fine with `&str`.  `TcpConnection::new()` needs `impl Into<String>` anyways

Reply:
I am guessing you meant it was fine with ```String```? Clippy complained that it wasn't consumed, but maybe that was because I had unecessary clones. Going to check again.

Reply:
Because I am logging it as the last usage, which doesn't consume, I think we need &str here.